File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/NuroOUT0101U02GridCommentInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable ]
	public class NuroOUT0101U02GridCommentInfo
	{
		private String _ser;
		private String _comment;
		private String _patientCode;

		public String Ser
		{
			get { return this._ser; }
			set { this._ser = value; }
		}

		public String Comment
		{
			get { return this._comment; }
			set { this._comment = value; }
		}

		public String PatientCode
		{
			get { return this._patientCode; }
			set { this._patientCode = value; }
		}

		public NuroOUT0101U02GridCommentInfo() { }

		public NuroOUT0101U02GridCommentInfo(String ser, String comment, String patientCode)
		{
			this._ser = ser;
			this._comment = comment;
			this._patientCode = patientCode;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/NuroOUT0101U02GridCommentInfo.java
package nta.med.data.model.ihis.nuro;

public class NuroOUT0101U02GridCommentInfo {
	private Double ser;
	private String comment;
    private String patientCode;
	  
	public NuroOUT0101U02GridCommentInfo(Double ser, String comment,
			String patientCode) {
		super();
		this.ser = ser;
		this.comment = comment;
		this.patientCode = patientCode;
	}
	
	public Double getSer() {
		return ser;
	}
	public void setSer(Double ser) {
		this.ser = ser;
	}
	public String getComment() {
		return comment;
	}
	public void setComment(String comment) {
		this.comment = comment;
	}
	public String getPatientCode() {
		return patientCode;
	}
	public void setPatientCode(String patientCode) {
		this.patientCode = patientCode;
	}
	  
}
